Update pull_request_template.md (#889)
bcgov/jag-file-submission: .github/pull_request_template.md

Please describe the tests that you ran to verify your changes.

> ## Does the change need to be reflected in Docker?

Reviewer: nit, could maybe say:
```suggestion
## Does the change impact or break the Docker build?
```

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes

> - [ ] No 

Reviewer: ```suggestion
- [ ] No
```
